lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -819,6 +821,20 @@ def __init__(self, verts, *args, zsort='average', **kwargs):
         zsort : {'average', 'min', 'max'}, default: 'average'
             The calculation method for the z-order.
             See `~.Poly3DCollection.set_zsort` for details.
+        shade : bool, default: False
+            Whether to shade *facecolors* and *edgecolors*.
+
+            .. versionadded:: 3.7
+
+            .. note::
+                *facecolors* and/or *edgecolors* must be provided for shading
+                to work.
```

Comment:
I think a note is more heavy than needed. We usually only state deprendencies between parameters in plain text:
```suggestion

            When activating *shade*, *facecolors* and/or *edgecolors* must be
            provided.

            .. versionadded:: 3.7
```

Reply:
Sure. Again this comes from the fact that ideally it should work without providing it, so I thought it was worth highlighting. But as it now errors as well, it is hard to miss it.